charts/camunda-platform-alpha/test/unit/web-modeler/golden/deployment-restapi.golden.yaml

```diff
@@ -64,6 +64,11 @@ spec:
                 secretKeyRef:
                   name: camunda-platform-test-postgresql-web-modeler
                   key: password
+            - name: RESTAPI_MAIL_PASSWORD
```

Comment:
@hamza-m-masood Could you check where this change came from? :grin:

Reply:
I remember there was a unit test for web-modeler failing because it checks for an existing string. I fixed this earlier on in the PR. I will add back the changes for the specific unit test.